### ISSUE TITLE
Datagrid: updateRow() drops the 'draggable' attribute on RowReorder Formatter

### ISSUE THREAD

**Issue by vonnyw** [infor-design/enterprise#4674]
**Describe the bug**
updateRow() drops the 'draggable' attribute of RowReorder Formatter, therefore the row is no longer draggable.
Additionally, the isHandle attribute is not evaluated appropriately within the arrange component.

**To Reproduce**
1. Add the following updateRow line to the datagrid-rowreorder.demo.ts:

```
[ 
 onRowReordered(event: SohoDataGridRowReorderedEvent) {
    console.log('startIndex: ' + event.startIndex, 'endIndex: ' + event.endIndex);
    this.datagrid.updateRow(event.endIndex, this.datagrid.dataset[event.endIndex]);
  }
]
```
![image](https://user-images.githubusercontent.com/39130590/101841206-1cdf3000-3b0b-11eb-9bf4-7b7a692367ad.png)

![image](https://user-images.githubusercontent.com/39130590/101841393-7182ab00-3b0b-11eb-9245-cfe8978283ab.png)

2. Go to http://localhost:4200/ids-enterprise-ng-demo/datagrid-rowreorder
3. Reorder a row
4. Try to drag the row up or down, it's no longer draggable.

**Expected behavior**
Drag handler should still work after updateRow is called.

**Version**

- ids-enterprise: 4.35

**Screenshots**

**Platform**
 - Infor Application/Team Name: Infor Landmark
 - OS Version: Windows 10
 - Browser Name: Chrome
 - Browser Version: Version 86.0.4240.198 (Official Build) (64-bit)

**Additional context**

**Comment by tmcconechy** [comment]
Please fix in 4.36 and I will patch in 4.35 once merged

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
